trunner/target/armv7a9.py

```diff
@@ -41,7 +41,12 @@ def _set_flash_mode(self, flash):
 
 
 class ZynqGdbPloLoader(TerminalHarness, PloInterface):
-    def __init__(self, dut: Dut, script: str, cwd: Optional[str] = None):
+    def __init__(
+        self,
+        dut: Dut,
+        script: str,
+        cwd: Optional[str] = None,
+    ):
```

Comment:
Why is it change? This line is too long for black? We want to call it with `-l 120`